src/chips/sam4l/target.json

```diff
@@ -6,5 +6,18 @@
     "arch": "arm",
     "os": "none",
     "morestack": false,
-    "executables": true
+    "executables": true,
+    "no-compiler-rt": true,
+    "relocation-model": "static",
+    "linker": "arm-none-eabi-gcc",
+    "disable-redzeon": true,
```

Comment:
heard, but what does this actually do? Works fine with this key mispelled, so maybe we don't need it? (side-note I would love to actually comment the target json, but I'm not entirely sure what most of the keys mean)